spec/features/local_auto_save_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
+
+describe 'Local auto save', js: true do
+  before { login_to_project_as_user }
+
+  let(:issue) { create(:issue, node: current_project.issue_library) }
+
+  let(:board) { create(:board, project: current_project, node: current_project.methodology_library) }
+  let(:list) { create(:list, board: board) }
+
+  let!(:users) do
+    user1 = create(:user, :author)
+    user2 = create(:user, :author)
+
+    [
+      user1,
+      user2
+    ]
+  end
+
+  describe 'custom tag input' do
+    let!(:tags) do
+      [
+        current_project.tags.create(name: '!9467bd_critical'),
+        current_project.tags.create(name: '!d62728_high')
+      ]
+    end
+
+    it 'will be cached' do
```

Comment:
I don't think the tense is consistent with the other assertions.

```suggestion
    it 'caches the content' do
```